ui.tests/test-module/specs/venia/react.js

```diff
@@ -45,39 +45,73 @@ describe('Venia React Components', () => {
         expect(minicart).toBeDisplayed();
     });
 
+    it('should redirect to homepage from address book page without login', () => {
+        // Go to address book page
+        browser.url(`${config.aem.author.base_url}/content/venia/us/en/my-account/address-book.html`);
+        browser.waitUntil(() => browser.getUrl() === venia_homepage, {
+            timeout: 5000,
+            timeoutMsg: 'expected browser to navigate to homepage after 5s'
+        });
+    });
+
+    it('should redirect to homepage from account info page without login', () => {
+        // Go to address book page
+        browser.url(`${config.aem.author.base_url}/content/venia/us/en/my-account/account-details.html`);
+        browser.waitUntil(() => browser.getUrl() === venia_homepage, {
+            timeout: 5000,
+            timeoutMsg: 'expected browser to navigate to homepage after 5s'
+        });
+    });
+
+    it('should redirect to homepage from order history page without login', () => {
+        // Go to address book page
+        browser.url(`${config.aem.author.base_url}/content/venia/us/en/my-account/order-history.html`);
+        browser.waitUntil(() => browser.getUrl() === venia_homepage, {
+            timeout: 5000,
+            timeoutMsg: 'expected browser to navigate to homepage after 5s'
+        });
+    });
+
     it('should render the sign in component in the header', () => {
         // Go to the Venia homepage
         browser.url(venia_homepage);
 
         // Check sign in button
-        const signInButton = $('.cmp-AccountContainer__accountTrigger__root');
-        expect(signInButton).toBeDisplayed();
+        const signInTrigger = $('.cmp-VeniaHeader__accountTrigger__trigger');
+        expect(signInTrigger).toBeDisplayed();
 
         // Check sign in form
-        signInButton.click();
-        const signInForm = $('.cmp-SignIn__signIn__form');
+        signInTrigger.click();
+        const signInForm = $('.cmp-VeniaSignIn__signIn__form');
         expect(signInForm).toBeDisplayed();
+
+        $('input[autocomplete="email"]').setValue(config.venia.email);
```

Comment:
what happens when the config.venia.email / .password is not set? (Like a user just clones the repo and runs the tests) 

I guess it will fail. If so could be skip the test when the config(s) are not provided?

Reply:
what about all the other configs?
See line 98

Reply:
fixed it